reactor-core/src/main/java/reactor/core/publisher/FluxRange.java

```diff
@@ -70,7 +70,8 @@ public void subscribe(CoreSubscriber<? super Integer> actual) {
 
 	@Override
 	public Object scanUnsafe(Attr key) {
-		return null; //no particular key to be represented, still useful in hooks
+		if (key == Attr.RUN_STYLE) return Attr.RunStyle.SYNC;
```

Comment:
`RangeSubscription` should also be covered

Reply:
Fixed in commit [26552f6](https://github.com/reactor/reactor-core/pull/2123/commits/26552f67f2cfd7c10e59e60f3b905504c55b6e1d)